src/utils/transformAwareTranslator.ts

```diff
@@ -1,103 +1,65 @@
-import * as d3 from "d3";
+/**
+ * Copyright 2017-present Palantir Technologies
```

Comment:
because the copyright didn't exist before and this is the first time i'm adding it.